packages/fether-react/src/utils/blockscout.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const baseUrl = chainName => {
+  let baseUrl;
```

Comment:
Can you pick a different variable name?

baseUrl is also the name of the function, when you do `baseUrl = ...` the reader is not 100% sure... It seems to work though.